cljfmt/test/cljfmt/core_test.cljc

```diff
@@ -24,15 +24,15 @@
   (testing "cond indentation"
     (is (= "(cond\n  a? a\n  b? b)"
            (reformat-string "(cond  \na? a\n   b? b)")))
-    (is (= "(cond\n  a?\n    a\n  b?\n    b)"
+    (is (= "(cond\n  a?\n  a\n  b?\n  b)"
```

Comment:
With these test changes, you're no longer exercising the `:stair` rule - maybe fix them up or add a separate block of tests for stair-formatted `cond`?